projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -669,6 +669,14 @@ export interface FeatureTogglesInterface {
    * Affects: `ReturnOrderComponent`
    */
   enableReturnOrderReturnableQuantityConsigmentFallback?: boolean;
+  /**
+   * When enabled, all cross-site requests will include credentials (cookies, HTTP authentication entries, and client-side SSL certificates).
+   * This is needed to support scenarios where the backend services are hosted on a different domain than the Spartacus storefront,
+   * and the authentication cookies need to be sent with the requests to maintain the user session.
+   *
+   * Affects: `WithCredentialsInterceptor`
+   */
+  enableWithCredentialsByDefault?: boolean;
```

Comment:
[nitpick]
This change affects only calls to OCC.
let's reflect it in the toggle name, e.g.:
`occWithCredentialsByDefault`

PS. I was never a fan of using the word `enable` in the toggles names. They should describe the change in behavior and that's it. By setting it to `true`, we just _enable_ it 😅

Reply:
Yea, fully agreed with this suggestion, thanks 👍🏼